src/libs/ErrorUtils.js

```diff
@@ -37,6 +38,16 @@ function getAuthenticateErrorMessage(response) {
     }
 }
 
+/**
+ * Method used to generate errors object optimistically.
+ * @param {String} error - error key or message to be saved
+ * @return {Object} - Object to be optimistically saved to Onyx
+ *
+ */
+function getOptimisticErrors(error) {
```

Comment:
Discussed this a bit with my team. We just found the method name to be a bit confusing. The method doesn't really give us optimisticData. It actually gives us failureData.

Can you rename this to,
```
getMicroSecondOnyxError
```